### DISCUSSION TITLE
how to add additional servo channels to MAMBA F405 for quadrocopter?

### DISCUSSION THREAD

**Discussion by Bookwormjim** [iNavFlight/inav#8389] [Q&A] [answered]
I have FC MAMBA F405 MK3 stack (MAMBA F405US_I2C), and I want to install  camera on a DIY gimbal on a servo.
But there are no S5/S6 pins for the servo on the FC, only S1/S4 pins for the motors.
I thought that it was easy to assign free pins or channels to the servo outputs, for example LED 1-2 or UART tx4/rx4. But I didn't find how to do it!
I tried to assign outputs via the CL "resource" command, but it only showed the pin map, does not allow them to be reassigned.
Please help!

**Comment by streem-ua** [comment]
Hello! @Bookwormjim Have you figured out how to connect a servo? I have mamba f722 mk2 and in same situation

**Comment by DzikuVx** [comment, marked as answer]
Mamba 405US does not have any outputs with free timer to be assigned as servos in MR mode. Same applies to Mamba F722
